roles/siege-install/tasks/main.yml

```diff
@@ -0,0 +1,48 @@
+---
+# taken from my courses siege-install.yml example
+- name: Create a Directory for the build
+  file:
+    path: /etc/siege/siege_build
+    state: directory
+    mode: 0644
+    seuser: system_u
+
+- name: Download Siege
+  get_url:
+    url: http://download.joedog.org/siege/siege-latest.tar.gz
+    dest: /etc/siege/siege_build
+    mode: 0744
+    seuser: system_u
+    force: true
+
+- name: unarchive
+  unarchive:
+    src: /etc/siege/siege_build/siege-latest.tar.gz
+    dest: /etc/siege/
+
+#Store the directory path in-order to change and set that directory for next part of installation
+
+- name: Installation Part-1
+  shell: "ls {{ item }}"
+  with_items:
+  - /etc/siege/
+  register: new
+
+# To display the output of the above command and
+# Due to multiple entries and "/etc/siege/siege-*/" unsupported, we use the following
+
+- debug:
+    var: new.results[0].stdout_lines[0]
+
+# These are the basic commands needed to install the necessary configuration and installation and creating the configuration file!
+
+- name: Installation Part-2
+  command: "{{ item }}"
+  with_items:
+  - ./configure
+  - make
+  - make install
+  - siege.config
+  args:
+    chdir: /etc/siege/{{ new.results[0].stdout_lines[0] }} #/etc/siege/siege-4.0.4/
+  register: all
```

Comment:
Ah, wasn't sure its a thing. The code in there was taken from the example file in mycourses.

Reply:
@BonjeBongo Ouch. I didn't realize the lab instructions did it this way. :see_no_evil: Let's just install the package from EPEL since the open source community has already gone through the trouble of packaging and installing this correctly as an RPM.